server/devdocs/bin/generate-components-usage-counts

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env node
+
+/**
+ * This script generates a usage counts for the dependecies of a list of modules.
+ * It accepts a newline-delimited list of .js and|or .jsx files
+ * as its input, and writes the index to server/devdocs/usage-counts.js
+ */
+
+var async = require( 'async' ),
+	fs = require( 'fs' ),
+	fspath = require( 'path' ),
+	root = fspath.dirname( fspath.join( __dirname, '..', '..' ) ),
+
+	// Copyright (c) 2014-present, Facebook, Inc. See CREDITS.md#L1383
```

Comment:
Yes indeed. I had the same thought when I saw how other files reference `CREDITS.md`.
Should I remove the line number?